packages/activity-streams.js/src/activity-streams.js

```diff
@@ -61,6 +61,7 @@ const ee = EventEmitter(),
 
 let objs = new Map(),
     failOnUnknownObjectProperties = false,
+    warnOnUnknownObjectProperties = true,
```

Comment:
Didn't we say it should not print to the console by default?

Reply:
When the server initializes the object, it disables the warnings here https://github.com/sockethub/sockethub/pull/737/files#diff-e1a9a5a584326eccba5e95ec2316a1484a3958a2eb01aa9aa8bf999eb7390baa